testing/src/scenario/state.py

```diff
@@ -9,12 +9,12 @@
 import dataclasses
 import datetime
 import inspect
+import pathlib
 import random
 import re
 import string
 from enum import Enum
 from itertools import chain
-from pathlib import Path, PurePosixPath
```

Comment:
Sad to see this change. I guess if it's the only way to make Sphinx happy ... I know we generally prefer to keep imports in their namespace, but
1. `from pathlib import Path` is so idiomatic
2. I kind of like seeing what's being used from an import in the imports
Ah well, not a blocker

Reply:
For what it's worth, I am very much on the side of it being an improvement anyway (a discussion for another day, I imagine) so I didn't look very hard at getting Sphinx to be happy with it as-is.

If you feel strongly about it, then I could try to see why Sphinx thought it was something defined in `scenario.state` and couldn't detect that it was coming from the standard lib.

Reply:
All good, I don't think it's worth spending further time on this. Keen to chat about import styles another day though